infrastructure/code/template/details.html

```diff
@@ -335,6 +346,45 @@
       position: absolute;
       padding: 0 4px;
     }
+
+    /* Footer Styles */
+    .ignore-action-container {
+      position: fixed;
+      bottom: 0;
+      width: 100%;
+      background-color: var(--vscode-editor-background);
+      background-image: linear-gradient(45deg, rgba(255, 255, 255, 0.075), rgba(255, 255, 255, 0.075));
+      box-shadow: 0 -1px 3px rgba(0, 0, 0, .05);
+    }
+
+    .actions {
+      justify-content: space-between;
+      display: flex;
+    }
+
+    .row {
+      display: flex;
+      flex-direction: row;
+      width: 100%;
+    }
+
+    .ignore-button.secondary {
+      cursor: pointer;
+      width: auto;
+      padding: 6px 16px;
+      border: 1px solid var(--vscode-textLink-foreground);
+      border-radius: 3px;
+      background: none;
+      line-height: 1;
+      color: var(--vscode-textLink-foreground);
+    }
+
+    .ignore-button.secondary:hover,
+    .ignore-button.secondary:active {
+      background: var(--vscode-button-hoverBackground);
+      border-color: var(--vscode-button-hoverBackground);
+      color: var(--vscode-button-foreground);
```

Comment:
These are going to be handled together with https://github.com/snyk/vscode-extension/pull/463